ARCHITECTURE.md

```diff
@@ -0,0 +1,78 @@
+# Architecture
+
+OpenPype is a monolithic Python project that bundles several parts, this document will try to give a birds eye overview of the project and, to a certain degreem, each of the sub-projects.
+The current file structure looks like this:
+
+```
+.
+├── common - Code in this folder is backend portion of Addon distribution logic for v4 server.
+├── docs - Documentation of the source code.
+├── igniter - The OpenPype bootstrapper, deals with creating necessary files and the connection to the mongodb.
```

Comment:
I would say that **igniter** helps with setting up connection to the mongo db and is dealing with OpenPype version resolution.
```suggestion
├── igniter - The OpenPype bootstrapper, deals with running version resolution and setting up the connection to the mongodb.
```